### PR TITLE
[us] How to polish code, eg: OC\User

### PR DESCRIPTION
**Pull request by rullzer**
from: https://github.com/owncloud/core/pull/25767

CC: @LukasReschke @nickvergessen @MorrisJobke @icewind1991

### CODE REVIEW

File: lib/private/User/Database.php [hunk 1]
```diff
@@ -55,17 +55,23 @@
 use OC\Cache\CappedMemoryCache;
 use Symfony\Component\EventDispatcher\EventDispatcher;
 use Symfony\Component\EventDispatcher\GenericEvent;
+use OCP\IUserBackend;
+use OCP\Util;
```
Comment (nickvergessen) [inline lib/private/User/Database.php]: Alphabet?

File: lib/private/User/Database.php [hunk 2]
```diff
@@ -55,17 +55,23 @@
...
+class Database extends Backend implements IUserBackend {
 	/** @var CappedMemoryCache */
 	private $cache;
+
 	/** @var EventDispatcher */
 	private $eventDispatcher;
+
 	/**
-	 * OC_User_Database constructor.
+	 * OC\User\Database constructor.
```
Comment (nickvergessen) [inline lib/private/User/Database.php]: Invalid name

**Comment by rullzer** [comment]
fixed

**Comment by nickvergessen** [comment]
:+1:

**Comment by LukasReschke** [comment]
LGTM

**Pull request merged**